cli/src/pcluster/config/cluster_config.py

```diff
@@ -191,8 +191,9 @@ def _register_validators(self):
 class RootVolume(Ebs):
     """Represent the root volume configuration."""
 
-    def __init__(self, delete_on_termination: bool = None, **kwargs):
+    def __init__(self, size: int = None, delete_on_termination: bool = None, **kwargs):
         super().__init__(**kwargs)
+        self.size = Resource.init_param(size)
```

Comment:
Why did you remove the default size from EBS and so from the RootVolume?

Reply:
Because the instance root volume will be created with the size of AMI automatically. It is a EC2 default. We don't need to specify the root volume size in our code